doc/CommonTraps.md

```diff
@@ -0,0 +1,32 @@
+# Common PowerPoint Add-in Development Traps
+
+This document details the common pitfalls when developing for PowerPointLabs, so that these mistakes can be noted/avoided by future contributors.
+
```

Comment:
Can also mention RestoreClipboardAfterAction is useful if your feature requires copy and pasting so that it will not affect the clipboard.

Reply:
Should we mention to convert from Shape to ShapeRange, there is a method called ToShapeRange in PowerPointSlide.cs

Reply:
1. need to consult @alexfjw on the inclusion of `#pragma warning disable 0618`. I personally have not had the need to include this, was wondering whether you feel that this is worth including. This is because the moment we include, people will definitely be more inclined to use it
1. Ill update to mention the RestoreClipboardAfterAction! Thanx for the heads up
1. Ill mention the method that converts Shape to ShapeRange as well

Reply:
I think we can say something a bit vague like:
Refrain from using #pragma warning disable 0618, unless necessary. I'm ambivalent about mentioning the places where it's used, as we might be overloading the newcomers with too much info.
A mention will help people will remember about it if they ever end up having a need to use it.

Reply:
Ive included a mention in the newcomer guide